### PR TITLE
COMMERCE-11209

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://issues.liferay.com/browse/COMMERCE-11209

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase [hunk 1]
```diff
@@ -1221,6 +1221,48 @@ definition {
...
 	}
 
+	@description = "This is a test for bug COMMERCE-11209. As the Admin I can delete a product image when the image is set via headless API."
+	@priority = 5
+	test CanDeleteProductImageWhenImageIsUploadedViaAPI {
+		task ("Given a Minium Site is created") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+		}
+
+		task ("And for the product ABS Sensor is uploaded an image using API") {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase:1231]:
```suggestion
		task ("And an image for ABS Sensor is uploaded via API") {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase [hunk 2]
```diff
@@ -1221,6 +1221,48 @@ definition {
...
+			var responseBody = CommerceJSONProductsAPI._patchCommerceProductImageUsingERC(
+				cdnEnabled = "true",
+				cdnURL = "http://mirrors.lax.liferay.com/github.com/liferay/liferay-portal/portal-web/test/functional/com/liferay/portalweb/dependencies/Commerce_Black.jpg",
+				imageTitle = "title2",
+				productERC = "MIN93015minium-initializer");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase:1236]: If `responseBody` is not used anywhere in the test, it is not necessary to have the responseBody returned

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase [hunk 3]
```diff
@@ -1221,6 +1221,48 @@ definition {
...
+			Click(
+				key_value = "title2",
+				locator1 = "CommerceEntry#BODY_VERTICAL_ELLIPSIS");
+
+			CommerceEntry.clickNoError(menuItem = "Delete");
+
+			AssertElementNotPresent(
+				key_entryName = "title2",
+				locator1 = "CommerceEntry#TABLE_LIST_TITLE");
+		}
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase:1263]:
```suggestion
		task ("And the image is successfully uploaded") {
			CommerceProducts.openProductsAdmin();

			CommerceProducts.searchAndGoToProductEntry(entryProductName = "ABS Sensor");

			CommerceEntry.gotoMenuTab(menuTab = "Media");

			AssertElementPresent(
				key_entryName = "title2",
				locator1 = "CommerceEntry#TABLE_LIST_TITLE");
		}

		task ("When the Admin tries to delete the image") {
			Click(
				key_value = "title2",
				locator1 = "CommerceEntry#BODY_VERTICAL_ELLIPSIS");

			CommerceEntry.clickNoError(menuItem = "Delete");
		}
		
		task ("Then the item is successfully removed") {
			AssertElementNotPresent(
				key_entryName = "title2",
				locator1 = "CommerceEntry#TABLE_LIST_TITLE");
		}
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase [hunk 4]
```diff
@@ -1221,6 +1221,48 @@ definition {
 			status = "Scheduled");
 	}
 
+	@description = "This is a test for bug COMMERCE-11209. As the Admin I can delete a product image when the image is set via headless API."
+	@priority = 5
+	test CanDeleteProductImageWhenImageIsUploadedViaAPI {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProducts.testcase:1226]:
```suggestion
	test CanDeleteProductImageUploadedViaAPI {
```

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Review by lorenzolapescara — APPROVED** [review]

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:forward

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by Brunno0liveira** [comment]
ci:forward

**Pull request closed without merging**